### DISCUSSION TITLE
Outil de recrutement pour les guildes

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#579] [Idées acceptées / déjà implémentées]
Création d’un tableau de recrutement des guildes avec la commande !g lorsque le joueur n’appartient à aucune, permettrait de remplir les guildes encore incomplètes et de faire recruter les joueurs désireux de rejoindre celles-ci.

**Comment by BastLast** [comment]
ya un channel qui a été créé sur le serveur du bot
